portfolio/src/main/webapp/index.html

```diff
@@ -4,16 +4,33 @@
     <meta charset="UTF-8">
     <title>3030</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
```

Comment:
nit: Oops it looks like rel=stylesheet is included twice now!

Reply:
Oh I scrolled past it so I hadn't seen it, my bad. Done!